docs/GraphQL/makingASimpleRequest.rst

```diff
@@ -1,4 +1,88 @@
 Making a simple request
 ============================
 
-TODO
\ No newline at end of file
+This guide is intended to give an example on how to make your first query and mutation on the platform. We will use the provided playground on `api.passi.is/playground <https://api.passi.is/playground>`_ to try our the requests to the graphQL server.
+
+We will assume that you have already have made your first pass template and wish to add new passes to it. We will also use the cookie provided from the UI to make the test request but there will be an example on how to use the api key in code examples. To generate a cookie to use, simply log on to `passi.is <https://passi.is>`_.
+
+What we first want to do is get the Id of the passtemplate we want to create the new pass with. To do this we will query the passTemplates to get a list of the passtemplates we have. 
```

Comment:
What we first want to do is get the Id of the passtemplate we want to create the new pass with.

change this to: ?
The first thing we want to do is get the Id of the passtemplate we want to create the new pass with.